core/cap-0067.md

```diff
@@ -385,6 +385,16 @@ contract: asset, topics: ["clawback", from:Address, sep0011_asset:String], data:
 
 
 #### Allow Trust / Set Trustline Flags
+If a trustline has the `AUTHORIZED_FLAG` added or removed, as a result of one of these operations, then a `set_authorized` event should be emitted.
+
+```
+contract: asset, topics: ["set_authorized", admin:Address, id:Address, sep0011_asset:String], data: authorize:bool
+```
+
+* `admin` is the issuer of the trustline asset.
```

Comment:
@leighmcculloch should we remove the admin from this event as well? It's not part of SEP-0041, but the admin isn't necessary on the event.

Reply:
Yes I think so.

Reply:
I think this change should capture what it does already.

Then in CAP-67 we change it.

The hard thing about this event is removing the admin changes the position of the id topic. Now is the time to make the change though.

Maybe in CAP-67 we should change the event's first topic too, from `set_authorized` to `authorized`. It's odd that it maps 1-1 with a function name. And changing the name would make it less bad to change the topic list ordering.

Reply:
My previous message is probably confusing because I thought this comment was on CAP-46-6.

Reply:
Wdyt:
```
contract: asset, topics: ["authorized", id:Address, sep0011_asset:String], data: authorized:bool
```

Reply:
Two separate events for authorization seems error prone, but so does changing the position of the id... I'm not sure which one is worse. I'm wondering if removing the admin is sufficient because we're also doing it to mint and clawback. Maybe consumers will seek out which events had the admin removed?